src/components/SimpleCommonComponents/Tooltip/style.css

```diff
@@ -10,6 +10,14 @@
   margin: 4px;
   z-index: var(--z-index-tooltips);
 
+  &.fixed {
```

Comment:
was this trying to fix the overgrown tooltip? is it still necessary after my fix?

Reply:
Yes. Feel free to remove it now that your fix addresses the overgrown tooltip.